packages/hawtio/src/core/config-manager.ts

```diff
@@ -236,6 +258,66 @@ class ConfigManager {
     }
     config.about.productInfo.push({ name, value })
   }
+
+  /*
+   * Synchronous method to retrieve the appearance.showHeader
+   * property from the existing hawtconfig. The hawtconfig
+   * should have been obtained using an async process,
+   * eg. the #useHawtconfig hook
+   */
+  isHeaderShown(hawtconfig: Hawtconfig): boolean {
+    if (!hawtconfig) return true
+
+    const appearance = hawtconfig.appearance
+    if (!appearance) return true
+
+    return appearance.showHeader ?? true
+  }
+
+  /*
+   * Synchronous method to retrieve the appearance.showBrand
+   * property from the existing hawtconfig. The hawtconfig
+   * should have been obtained using an async process,
+   * eg. the #useHawtconfig hook
+   */
+  isBrandShown(hawtconfig: Hawtconfig): boolean {
+    if (!hawtconfig) return true
+
+    const appearance = hawtconfig.appearance
+    if (!appearance) return true
+
+    return appearance.showBrand ?? true
+  }
+
+  /*
+   * Synchronous method to retrieve the appearance.showUserHeader
+   * property from the existing hawtconfig. The hawtconfig
+   * should have been obtained using an async process,
+   * eg. the #useHawtconfig hook
+   */
+  isUserHeaderShown(hawtconfig: Hawtconfig): boolean {
+    if (!hawtconfig) return true
+
+    const appearance = hawtconfig.appearance
+    if (!appearance) return true
+
+    return appearance.showUserHeader ?? true
+  }
+
+  /*
+   * Synchronous method to retrieve the appearance.showSideBar
+   * property from the existing hawtconfig. The hawtconfig
+   * should have been obtained using an async process,
+   * eg. the #useHawtconfig hook
+   */
+  isSideBarShown(hawtconfig: Hawtconfig): boolean {
+    if (!hawtconfig) return true
+
+    const appearance = hawtconfig.appearance
+    if (!appearance) return true
+
+    return appearance.showSideBar ?? true
+  }
```

Comment:
These `isXxx` methods don't add any values so not necessary. We can just chain the fields with `xxx?.yyy` in one line. Please remove them. The design of the methods is not cool as well, as they accepts `hawtconfig` while it should be already there in the object field. This aspect further clutters the interface API.